src/app/store/services/shared.service.ts

```diff
@@ -17,6 +17,48 @@ import { PlanType, PricingPlan } from '../datatypes';
 import { MailContactComponent } from '../../mail/mail-contact/mail-contact.component';
 import { SaveContactComponent } from '../../mail/mail-contact/save-contact/save-contact.component';
 
+import Quill from 'quill';
+import ImageResize from 'quill-image-resize-module';
+
+
+// Register quill modules and fonts and image parameters
+Quill.register('modules/imageResize', ImageResize);
```

Comment:
you shouldn't register it here, both register events should be done in component, this should be moved back.
```
Quill.register('modules/imageResize', ImageResize);
```

Reply:
Okay